### ISSUE TITLE
Autocomplete options are not accessible to the user on JAWS + Firefox

### ISSUE THREAD

**Issue by calebegg** [angular/material#6580]
Only the first option can be selected on JAWS, using the autocomplete demo page, for example. The later options are not selectable.

**Comment by ThomasBurleson** [comment]
This issue is closed as part of our deprecation effort. 
      For details, see our post [Spring Cleaning 2016](https://groups.google.com/forum/#!topic/ngMaterial/3Ip9nd68bfk).
